### PR TITLE
Add a news post for `osu!CenterStage - Idol 2021`

### PR DESCRIPTION
**Pull request by MjIsMaster**
This is a news post for the auditions of osu!CenterStage (official successor to osu!idol)

### CODE REVIEW

File: news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "osu!CenterStage - Idol 2021"
+date: 2021-07-02 12:00:00 +0000
+---
+
+We are happy to announce the yearly idol contest is back and ready for action!
+![](/wiki/shared/news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open/centerstagebanner.jpg)
+A new chapter in the history of this competition, but the spirit is as strong as ever!
```
Comment (TPGPL) [inline news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md:9]:
```suggestion
We are happy to announce the yearly idol contest is back and ready for action!

![](/wiki/shared/news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open/centerstagebanner.jpg)

A new chapter in the history of this competition, but the spirit is as strong as ever!
```

File: news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+### **What is osu!CenterStage?**
```
Comment (TPGPL) [inline news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md:11]:
```suggestion
## What is osu!CenterStage?

```

File: news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+A successor to the much beloved osu!idol competition which concluded in 2020. The format remains largely the same. Participants will sing through the various rounds set for them until a winner is decided. A panel of judges will guide the direction of this contest by evaluating and critiquing the participant's performances.
+
+### **How does this work?**
```
Comment (TPGPL) [inline news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md:14]:
```suggestion
## How does this work?

```

File: news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+For the auditions, record yourself performing a 30-60 second sample of any song (with no BGM) then submit in your entry. It's as simple as that! July 21st will be the deadline so make haste and submit in those wonderful entries!
+
+### **Any Prizes?**
```
Comment (TPGPL) [inline news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md:17]:
```suggestion
## Any prizes?

```

File: news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+Why yes we do. We have some rewards in store for those who come out on top.
+
+- **Winner** - osu!supporter tag for 8 months
+- **Runner-Up** - osu!supporter tag for 6 months
+- **Third Place** - osu!supporter tag for 4 months
+
+For all relevant information you will need to know in entering this year's competition check out the [*official thread*](https://osu.ppy.sh/community/forums/topics/1359674). To get involved with the community and receiving important updates regarding this contest, join the official [*discord server*](https://discord.gg/eBaDTYrvby).
```
Comment (TPGPL) [inline news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md:24]:
```suggestion
For all relevant information you will need to know in entering this year's competition check out the [official thread](https://osu.ppy.sh/community/forums/topics/1359674). To get involved with the community and receiving important updates regarding this contest, join the official [Discord server](https://discord.gg/eBaDTYrvby).
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by MjIsMaster — COMMENTED** [review]
All suggestions have been committed.

File: news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "osu!CenterStage - Idol 2021"
+date: 2021-07-02 12:00:00 +0000
+---
+
+We are happy to announce the yearly idol contest is back and ready for action!
```
Comment (TPGPL) [inline news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md:7]:
```suggestion
We are happy to announce that the yearly idol contest is back and ready for action!
```

File: news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+![](/wiki/shared/news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open/centerstagebanner.jpg)
+
+A new chapter in the history of this competition, but the spirit is as strong as ever!
+
+## What is osu!CenterStage?
+
+A successor to the much beloved osu!idol competition which concluded in 2020. The format remains largely the same. Participants will sing through the various rounds set for them until a winner is decided. A panel of judges will guide the direction of this contest by evaluating and critiquing the participant's performances.
```
Comment (TPGPL) [inline news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md:15]:
```suggestion
A successor to the much beloved osu!idol competition which concluded in 2020. The format remains largely the same. Participants will sing through the various rounds set for them until a winner is decided. A panel of judges will guide the direction of this contest by evaluating and critiquing the participants' performances.
```

File: news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+## How does this work?
+
+For the auditions, record yourself performing a 30-60 second sample of any song (with no BGM) then submit in your entry. It's as simple as that! July 21st will be the deadline so make haste and submit in those wonderful entries!
```
Comment (TPGPL) [inline news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md:19]:
```suggestion
For the auditions, record yourself performing a 30-60 second sample of any song (with no background music), then submit in your entry. It's as simple as that! July 21st will be the deadline, so make haste and submit in those wonderful entries!
```

File: news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+## Any prizes?
+
+Why yes we do. We have some rewards in store for those who come out on top.
+
+- **Winner** - osu!supporter tag for 8 months
+- **Runner-Up** - osu!supporter tag for 6 months
+- **Third Place** - osu!supporter tag for 4 months
+
+For all relevant information you will need to know in entering this year's competition check out the [official thread](https://osu.ppy.sh/community/forums/topics/1359674). To get involved with the community and receiving important updates regarding this contest, join the official [Discord server](https://discord.gg/eBaDTYrvby).
```
Comment (TPGPL) [inline news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md:29]:
```suggestion
For all relevant information you will need to know before entering this year's competition, check out the [official thread](https://osu.ppy.sh/community/forums/topics/1359674). To get involved with the community and receive important updates regarding this contest, join the official [Discord server](https://discord.gg/eBaDTYrvby "Discord").
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by MjIsMaster — COMMENTED** [review]
Additional suggestions have been committed.

**Review by TPGPL — DISMISSED** [review]
LGTM; if you plan to make any more changes on your local side, use `git fetch origin` and then `git reset --hard origin/osu!CenterStage`, so the commit history doesn't break

**Comment by MjIsMaster** [comment]
Added a few words at the end of the post and fixed dates to match. Everything else remains the same

**Comment by roansong** [comment]
I see the registration deadline is August 1st now on the forum post; I think this iteration of the news post isn't going to make it.

**Comment by TPGPL** [comment]
@MjIsMaster is this news post still relevant or can the PR be closed safely?

**Comment by TPGPL** [comment]
sadly it looks like the PR is outdated, so I will close it for now

**Pull request closed without merging**